prime-router/docs/universal-pipeline/receive.md

```diff
@@ -1,11 +1,32 @@
 # Receive
+`Receive` is the first step a message goes through when entering the Universal Pipeline (UP). Despite its name, this step
+actually occurs in the `ReportFunction.kt` file. 
 
+There are two calls that will kick off this function:
+```
+POST /api/reports
+POST /api/waters
+```
```

Comment:
Should there be info on expected parameters in the calls?

Reply:
Is this necessary? I feel like we should just point to the swagger doc (doesn't exist yet). If you add it here, then you have to keep it up to date in multiple places instead of having a single source of truth.